static/src/javascripts/projects/common/modules/identity/modules/modal.js

```diff
@@ -4,6 +4,35 @@ import fastdom from 'lib/fastdom-promise';
 const ERR_MODAL_NOT_FOUND = 'Modal not found';
 const ERR_MODAL_MALFORMED = 'Modal is malformed';
 
+const removeNewsletterReminders = (modalEl: HTMLElement) => {
+    const newsletterReminders = modalEl.querySelector(
+        `.identity-consent-journey-modal-reminder`
+    );
+    if (newsletterReminders != null) {
+        while (newsletterReminders.firstChild) {
+            newsletterReminders.removeChild(newsletterReminders.firstChild);
+        }
+    }
+};
+
+const insertNewsletterReminders = (
+    modalEl: HTMLElement,
+    uncheckedNewsletters: Array<string>
+): Promise<HTMLElement> =>
+    fastdom.write(() => {
+        uncheckedNewsletters.map(newsletterName => {
+            const li = document.createElement('li');
+            li.innerHTML = `<b>${newsletterName}</b>`;
+            const reminderEl = modalEl.querySelector(
+                `.identity-consent-journey-modal-reminder`
+            );
+            if (reminderEl != null) {
```

Comment:
same comment as before on this `if`.  Avoid `!=` if you can, `!==` would be the way to go if you have to.